src/08_comprehensions.py

```diff
@@ -10,15 +10,17 @@
 
 # Write a list comprehension to produce the array [1, 2, 3, 4, 5]
 
-y = []
-
+y = [x for x in range(1, 6)]
 print (y)
 
 # Write a list comprehension to produce the cubes of the numbers 0-9:
 # [0, 1, 8, 27, 64, 125, 216, 343, 512, 729]
 
 y = []
 
+for num in range(10):
```

Comment:
You need to do it as a list Comprehension. So, something like -` y = [x ** 3 for x in range(0, 10)]`